Makefile

```diff
@@ -105,7 +105,7 @@ manifest-lint: ## Run manifest validation
 build: generate manifests manager tools ## Build everything
 
 .PHONY: manager
-manager: generate lint ## Build manager binary
+manager: ## Build manager binary
```

Comment:
I think `generate` is quite important? I.e. now `make manager` may not produce valid results any more.

Reply:
I think this is a philosophical question.  Is code generation part of producing a binary, or is it a code change?